examples/webpack.config.local.js

```diff
@@ -34,6 +36,12 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
         'nebula.gl/dist': MAIN_SRC_DIR,
         'nebula.gl': MAIN_SRC_DIR,
 
+        '@nebula.gl/core/dist': CORE_SRC_DIR,
+        '@nebula.gl/core': CORE_SRC_DIR,
+
+        '@nebula.gl/geojson-modes/dist': GEOJSON_MODES_SRC_DIR,
```

Comment:
Are these EditModes so big that you need to offer multiple modules for edit modes? If not, just call this `edit-modes` for now?

Reply:
Good idea! KISS